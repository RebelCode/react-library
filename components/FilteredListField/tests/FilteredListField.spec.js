import React from "react";
import { render, cleanup } from "@testing-library/react";
import { FilteredListField } from "../index";

afterEach(cleanup);

describe("FilteredListField", () => {
  it("should take a snapshot", () => {
    const { asFragment } = render(<FilteredListField />);

    expect(asFragment(<FilteredListField />)).toMatchSnapshot();
  });

  it("renders correctly", () => {
    const { queryByLabelText } = render(<FilteredListField />);

    expect(queryByLabelText("ListField")).toBeTruthy();
  });
});
