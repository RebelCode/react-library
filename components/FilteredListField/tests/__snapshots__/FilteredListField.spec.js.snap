// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilteredListField should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="react-select css-dyr6gj-container"
  >
    <span
      aria-atomic="false"
      aria-live="polite"
      aria-relevant="additions text"
      class="css-1f43avz-a11yText-A11yText"
    />
    <div
      class="react-select__control css-iq6f7g-control"
    >
      <div
        class="react-select__value-container react-select__value-container--is-multi css-1vglxg7-ValueContainer"
      >
        <div
          class="react-select__placeholder css-1wa3eu0-placeholder"
        >
          Type something and press enter...
        </div>
        <div
          class="css-1epxsz4-Input"
        >
          <div
            class="react-select__input"
            style="display: inline-block;"
          >
            <input
              aria-autocomplete="list"
              aria-label="ListField"
              autocapitalize="none"
              autocomplete="off"
              autocorrect="off"
              id="react-select-2-input"
              spellcheck="false"
              style="box-sizing: content-box; width: 2px; border: 0px; opacity: 1; outline: 0; padding: 0px;"
              tabindex="0"
              type="text"
              value=""
            />
            <div
              style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
            />
          </div>
        </div>
      </div>
      <div
        class="react-select__indicators css-1hb7zxy-IndicatorsContainer"
      />
    </div>
  </div>
</DocumentFragment>
`;
